core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -338,7 +338,7 @@ class Executor[S <: Scheduling] private[cuttle] (val platforms: Seq[ExecutionPla
 
   private val pausedState: TMap[String, Map[Execution[S], Promise[Completed]]] = {
     val byId = TMap.empty[String, Map[Execution[S], Promise[Completed]]]
-    val pausedIds = queries.getPausedJobIds.transact(xa).unsafeRunSync
+    val pausedIds = Try { queries.getPausedJobIds.transact(xa).unsafeRunSync }.getOrElse(Set.empty)
```

Comment:
Why this would fail? Do we want to lose exceptions there?

Reply:
Doobie Transactor is especially annoying to mock in unit tests but I will try and see what I can do about it.